tskv/src/kvcore.rs

```diff
@@ -499,6 +508,10 @@ impl Engine for TsKv {
         while let Some(_x) = rx.recv().await {
             continue;
         }
+        self.sync_indexs()
+            .await
+            .expect("Tskv Index haven't been sync.");
+
         info!("TsKv closed");
```

Comment:
ts_index: Arc<RwLock<TSIndex>>,
ts_family: Arc<RwLock<TseriesFamily>>,
close the both！